test/AccessControlTests/APITests.cpp

```diff
@@ -0,0 +1,171 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include <Security.AccessControl.h>
+#include <winrt/Microsoft.Windows.Security.AccessControl.h>
+
+using namespace WEX::Common;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+namespace winrt
+{
+    using namespace winrt::Windows::ApplicationModel::Activation;
+    using namespace winrt::Windows::Foundation;
+    using namespace winrt::Windows::Foundation::Collections;
+    using namespace Microsoft::Windows::Security::AccessControl;
+}
+
+
+namespace Test::AccessControl
+{
+    class APITests
+    {
+        wil::unique_process_handle m_processHandle;
+        winrt::com_ptr<IApplicationActivationManager> m_testAppLauncher;
+
+        static PCWSTR GetTestPackageFile()
+        {
+            return L"AccessControlTestAppPackage";
+        }
+
+        static PCWSTR GetTestPackageFullName()
+        {
+            return L"AccessControlTestPackage_1.0.0.0_" WINDOWSAPPRUNTIME_TEST_PACKAGE_DDLM_ARCHITECTURE "8wekyb3d8bbwe";
+        }
+
+        static PCWSTR GetTestPackageFamilyName()
+        {
+            return L"AccessControlTestAppPackage_8wekyb3d8bbwe";
+        }
+
+        static PCWSTR GetTestPackageAppId()
+        {
+            return L"AccessControlTestAppPackage_8wekyb3d8bbwe!App";
+        }
+
+        void RunTestApp()
+        {
+            DWORD processId{};
+            VERIFY_SUCCEEDED(m_testAppLauncher->ActivateApplication(GetTestPackageAppId(), L"", AO_NONE, &processId));
+
+            m_processHandle.reset(::OpenProcess(SYNCHRONIZE | PROCESS_QUERY_LIMITED_INFORMATION, FALSE, processId));
+            VERIFY_IS_TRUE(m_processHandle.is_valid());
+        }
+
+    public:
+        BEGIN_TEST_CLASS(APITests)
+            TEST_CLASS_PROPERTY(L"Description", L"Windows App SDK AccessControl test")
+            TEST_CLASS_PROPERTY(L"ThreadingModel", L"MTA")
+            TEST_CLASS_PROPERTY(L"RunAs:Class", L"RestrictedUser")
+        END_TEST_CLASS()
+
+        TEST_CLASS_SETUP(ClassInit)
+        {
+            try
+            {
+                ::Test::Packages::RemovePackage(GetTestPackageFamilyName());
+                ::Test::Bootstrap::Setup();
+                ::Test::Packages::WapProj::AddPackage(TAEF::GetDeploymentDir(), GetTestPackageFile(), L".msix"); // Installs PushNotificationsTestApp.msix
```

Comment:
Do you need it?

Reply:
Is this about the out-of-date comment? I've updated it.